### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by wrycu** [UOAF/project-tauntaun#141]
None of the documentation explains what `map_token` in the config file is for or how to set it.

For anyone else looking, you go to https://console.cloud.google.com/projectselector2/google/maps-apis/credentials, create a new project, and create an API token, then use that as the value.

**Comment by adamdoda** [comment]
It's obsolete now, we use the legacy esri maps which are free no need for a mapbox account.

**Comment by adamdoda** [comment]
Good point I'll add a dummy release with the info to check the actions tab.
